src/test/java/core/basesyntax/CsvFileReaderServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.serviceimpl.CsvFileReaderServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvFileReaderServiceTest {
+    private static CsvFileReaderServiceImpl csvFileReaderService;
+
+    @Before
```

Comment:
'Before' will create new instance of service for each test. It's better to use 'BeforeClass' annotation. Fix it in other places